src/frontend/nginx.conf

```diff
@@ -1,32 +1,25 @@
-events {
-  worker_connections  256;
-}
+worker_processes 4;
+
+events { worker_connections 1024; }
```

Comment:
how it will help here?

Reply:
Sets the maximum number of simultaneous connections that can be opened by a worker process.
http://nginx.org/en/docs/ngx_core_module.html#worker_connections